docs/multinode-metrics.md

```diff
@@ -1 +1,34 @@
+## Metrics to enhance the grafana for multi-node etcd
+
+
+#### Etcd
```

Comment:
What about the following metrics?

- `etcd_network_peer_round_trip_time_seconds`
- `etcd_server_heartbeat_send_failures`
- `etcd_server_slow_apply_total`
- `etcd_server_slow_read_indexes_total`

Reply:
This is just a PR, motive is to get start working on metrics with basic metrics, We will keep adding whatever we feel needs to added or needed to expose.

`etcd_network_peer_round_trip_time_seconds` 
As Shesha and Madhav worked with etcd network traffic and I guess if they feel it will be helpful in fine tuning the zonal cluster then we can add. I don't have a strong opinion on this, I will add this if they feel it should be added.
cc @unmarshall  

`etcd_server_heartbeat_send_failures`
What information can we get from this ? 
there will be a hearbeat failures which will result into leader-election may be but that's a normal scenario isn't ? especially in etcd zonal cluster.

Reply:
It'll be useful if we see an unexpected amount of leader election and double checking `etcd_server_heartbeat_send_failures` can help to find the root cause for it.

Reply:
added

Reply:
`etcd_server_slow_apply_total`, `etcd_server_slow_read_indexes_total` not sure about these 2 metrics but I will add them as well ... no harm in it.

Reply:
added.